addon/evaluate-conditions.js

```diff
@@ -1,4 +1,11 @@
+/**
+ * @module evaluate-conditions
```

Comment:
Do we have a way of preventing imports of modules we intend to be private? If there isn't then it's kind of unfair to us to have to support usage we don't expose in the public API.